about.html

```diff
@@ -1,57 +1,58 @@
 <!DOCTYPE html>
-<html> 
+<html>
 <head>
-    <title>JumboCode: About</title>
-    <meta charset="utf-8">
-    <meta name="description" content="About Us Page of the JumboCode website">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-</head>
-
-<body>
-
-    <div>
-        <h2>Who We Are</h2>
-    </div>
-
-    <div>
-        <h2>How We Operate</h2>
-    </div>
-
-    <div>
-        <h2>Executive Board</h2>
-        <img src="image location" />
-        <h3>Yuki Zaninovich</h3>
-        <p>President</p>
 
-        <img src="image location" />
-        <h3>Hayley Cohen</h3>
-        <p>Client Manger</p>
+	<title>Jumbocode: About</title>
+	<meta charset="UTF-8">
+  	<meta name="description" content="About JumboCode">
+  	<meta name="keywords" content="JumboCode, About, Nonprofit">
 
-        <img src="image location" />
-        <h3>Andrew Hoiberg</h3>
-        <p>Project Manger</p>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-color/2.1.2/jquery.color.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/svg.js/2.6.3/svg.js"></script>
 
-        <img src="image location" />
-        <h3>Ben Cresitello-Dittmar</h3>
-        <p>Project Manger</p>
+	<link rel="stylesheet" href="styles.css">
+	<script src="scripts.js"></script>
 
-        <img src="image location" />
-        <h3>ShanShan Duan</h3>
-        <p>Social Media and Marketing</p>
-
-        <img src="image location" />
-        <h3>Becky Cutler</h3>
-        <p>Operations Manager</p>
-
-        <img src="image location" />
-        <h3>Emily Lin</h3>
-        <p>Design Lead</p>
-
-        <img src="image location" />
-        <h3>Spencer Perry</h3>
-        <p>Operations Manager</p>
-    </div>
+</head>
+<body>
+	<div class="navigation">
+		<ul>
+			<li><a href="contact.html">Contact</a></li>
+			<li><a href="projects.html">Projects</a></li>
+			<li><a href="about.html">About Us</a></li>
+			<li class="logo"><a href="index.html">JumboCode</a></li>
+		</ul>
+	</div>
+	<div class="page-content">
+		<div class="main-quote">
+			<h1>Philanthropy,</h1>
+			<h1>meet the <span class="green">future</span></h1>
+		</div>
+
+		<div class="explanations">
+			<div class="flex-grid row">
+				<div class="col explain-t">
```

Comment:
`explain-t` and `explain-p` are too ambiguous, please choose better names - it is okay if they are a little longer